pkgs/by-name/do/doomsday-engine/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  callPackage,
+  fetchFromGitea,
+
+  glbinding,
+  assimp,
+  qt5,
+  xorg,
+}:
+let
+  version = "2.3.1";
+in
+callPackage (import ./generic.nix {
```

Comment:
```suggestion
callPackage ./generic.nix {
```

Reply:
This was a suggestion made by @getchoo — clearly a consensus has to be made here on what should be the right pattern to have here